nixos/modules/services/web-apps/freshrss.nix

```diff
@@ -243,6 +244,7 @@ in
         in
         {
           description = "Set up the state directory for FreshRSS before use";
+          after = [ "network.target" ] ++ lib.optional usePostgresql "postgresql.service" ++ lib.optional useMysql "mysql.service";
```

Comment:
If you want to make sure to connect the Internet, you need to use:

```
wants = [ "network-online.target" ];
after = [ "network-online.target" ];
```

see `man systemd.special`

`usePostgresql` and `useMysql` only used once, no need to make variables for them

`lib.` can be omitted

Reply:
>  to ensure doesn't fail on database upgrades

If you want this service `freshrss-config` (and probably also `freshrss-updater`) to be restarted after database is restarted, `after` is not enough.  Check `bindsTo` or similar things.

Reply:
Thanks for the comments, I'll make the changes :)

Reply:
Alrighty, updated :)